tests/e2e/fill-incorrect-identity-input-values.test.ts

```diff
@@ -34,7 +34,7 @@ describe("Account Web Application add identity", () => {
   });
 
   test("should show error messages if password inputs are filled incorrectly", async (done) => {
-    expect.assertions(8);
+    expect.assertions(7);
```

Comment:
The failed results of e2e tests in CI shows that it received 8 assertions. Maybe you should put back 8 here 🙂